README.md

```diff
@@ -34,7 +34,6 @@ This package will come with:
   * [@nuxtjs/module-test-utils](https://www.npmjs.com/package/@nuxtjs/module-test-utils) - Additional testing utilities for *modules*
   * [@vue/test-utils](https://www.npmjs.com/package/@vue/test-utils) - Test utils for Vue components
   * [jsdom-global](https://www.npmjs.com/package/jsdom-global) - Required by @vue/test-utils
```

Comment:
Hi @Skasi , thanks for the PR and finding the duplicate. The two entries were originally supposed to be 'jsdom' and 'jsdom-global', but since jsdom-global (and this package) include jsdom as a devDependency, the single bullet should be fine.